### DISCUSSION TITLE
Thank you for the HiDPI/High Resolution support!!  :)

### DISCUSSION THREAD

**Discussion by Audiojunkie** [monocasual/giada#618] [General]
Thank you for the HiDPI/High Resolution support!!  I haven't tested it yet, but I had been waiting for it because I use a 4k laptop. I just noticed that it is now done. Thank you so much!!  :)

**Comment by gvnnz** [comment]
Thank you for your kind words, @Audiojunkie ! We are slowly approaching to Giada [version 1.0](https://github.com/monocasual/giada/milestone/4), more sweetness is about to come.
